app/components/NestedCheckboxes.tsx

```diff
@@ -57,6 +57,9 @@ function NestedCheckboxNode({
       }
     }
   })
+
+  const isExternalLink = link && link.startsWith('http')
```

Comment:
Technically, there are many href values that start with http that would _not_ be external links: `http`, `http/foo/bar`, etc. Take a look at how [react-router](https://github.com/remix-run/react-router)'s Link component or [rehype-external-links](https://github.com/rehypejs/rehype-external-links) do it.

Reply:
I updated this in the latest commit to compare the provided path with our origin. I'm reading over the react-router link now though, that might have a better implementation